download_file.py

```
@@ -1,22 +1,47 @@
-
+##########################################################################
+###################### First Latvian Fasker' Ripper ######################
+#################### Copyright (c) 2024-2025 mr.Iceman ###################
+##########################################################################
 from urllib.request import urlopen, Request
 from PIL import Image
 import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)